exercises/Laisha-Calalpa/answers.js

```diff
@@ -0,0 +1,121 @@
+// Question 1
+function greet(name) {
+  return(`Good Morning ${name}`);
+}
+
+// Question 2
+function logOddNumbers() {
+  for(i = 1; i <= 99; i++){
+    console.log(i);
+  }
+}
+
+// Question 3
+function area(length, width) {
+  rect_area = length * width;
+  return(rect_area);
+}
+
+// Question 4
+function otherAngle(first_angle, second_angle){
+  third_angle = 180 - (first_angle + second_angle);
+  return(third_angle);
+}
+
+// Question 5
+function diameterFromArea(area){
+  diameter = 2 * (Math.sqrt((area/Math.PI)));
+  return(diameter);
+}
+
+// Question 6
+function totalPrice(bill, tax){
+  total_price = bill + (bill * (tax/100));
+  return(total_price);
+}
+
+// Question 7
+function shortLongShort(first, second) {
+  if (first.length > second.length){
+    return(second + first + second)
+  }
+  else{
+    return(first + second + first)
+  }
+}
+
+// Question 8
+function describeAge(age){
+  let message = "";
+  message = (age <= 12) ? "You're a kid": (age >= 13 && age <=17) ? "You're a teenager" :  (age >= 18 && age <= 64) ? "You're an adult":  (age >= 65) ? "You're an elderly": false
```

Comment:
1. Don't declare `message`. Unnecessary.
2. Take advantage of the fact that ternary operators read from left to right by getting rid of every `&&`.